docs/server/README.md

```diff
@@ -1,5 +1,16 @@
 # Discodo Server Documentation
 
+## Urls
```

Comment:
`--port PORT, -P PORT  the port of the webserver (default: 8000)`

Looks like the default port is **8000**

Reply:
@wonderlandpark  
ex)
`--port PORT, -P PORT the port of the webserver (default: 8000)`
```
https://localhost:8000/
```

this like?

Reply:
I think it should be http

Reply:
I think Endpoints better than Urls

Reply:
> @wonderlandpark
> ex)
> `--port PORT, -P PORT the port of the webserver (default: 8000)`
> 
> ```
> https://localhost:8000/
> ```
> 
> this like?

It would be better to use the format <host>:<port>/ than localhost.

Reply:
How about BaseURL

Reply:
Ok BaseUrls